test/Unit/OptionTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Prewk\Option\Test\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Prewk\Option;
+use Prewk\Option\None;
+use Prewk\Option\Some;
+
+/**
+ * @covers \Prewk\Option
+ * @uses \Prewk\Option\None
+ * @uses \Prewk\Option\Some
```

Comment:
question: What are these for? Coveralls?

Reply:
Yes, and for infection to know what to do.

tl;dr `@covers` is for code that should be included in coverage reports, and modified by infection. `@uses` is for code used by the tests, but to not be included in coverage reports, or modified by infection.